### ISSUE TITLE
[RFE] Education - Send students for “local” education in other systems

### ISSUE THREAD

**Issue by psikomonkie** [MegaMek/mekhq#5484]
### Prerequisites and Pre-Issue Checklist

- [X] I'm reporting the RFE to the correct repository:
- [MegaMek](https://github.com/MegaMek/megamek/issues)
- [MegaMekLab](https://github.com/MegaMek/megameklab/issues)
- [MekHQ](https://github.com/MegaMek/mekhq/issues)

- [X] Does this conform with our Content Policy linked above?
- [X] I've searched the GitHub tracker and haven't found a similar feature request listed

### RFE Details *

This is an improvement to an existing feature

### Brief Description *

Allow us to send students to other systems for that system’s “local” education. Ideally it’ll default to our homeworld.

derrickthefirst on Discord suggested that it be possible to send students for “standard” education at our homeworld. 

I also don’t like sending students to local education when on an offensive contract, so this provides an option beyond prestige or in-company training. 

### MekHQ Suite Version *

Latest

### Final Checklist

- [X] I've checked to make sure that this RFE has not already been filed
- [X] I am requesting implementation of only one relevant set of features in this ticket for clarity and focus

**Comment by LadyAdia** [comment]
We could possibly reuse the prestigious education implementation for this, but I'd need someone more familiar with the code to weigh in on it.

**Comment by IllianiCBT** [comment]
This would require a significant rework of how we parse what academies are available. As _all_ Local Academy academies are available in all systems with a population allowing users to expand their search beyond current system would increase load exponentially.

**Comment by psikomonkie** [comment]
> This would require a significant rework of how we parse what academies are available. As _all_ Local Academy academies are available in all systems with a population allowing users to expand their search beyond current system would increase load exponentially.

I see the issue, I was hoping it'd be possible to only load the local academy list for whatever systems we're interested in.

**Comment by IllianiCBT** [comment]
Yeah, the curriculum picker is a hack and not how I would have implemented it today.

**Comment by HobbitDerrick** [comment]
My stupid and lazy idea for this was:

Add "Homeworld" to the unit definition.
Set that to Galatea at campaign start.
When the Unit is not under Contract, the Homeworld can be changed through the Interstellar map.
Local Academy education of dependents (only) is diverted to the Homeworld.

I think this handles all of what psikomonkie wanted. I hope it's simple but maybe it's not - I haven't read the education module code, but from the outside it seems to save the location of the Local Academy provider when a student is assigned to a course?
